tests/test_regions.py

```diff
@@ -1,81 +1,70 @@
+import functools
+
 import pytest
+from _pytest.monkeypatch import MonkeyPatch
 from boto3.session import Session
 
 from botocove import cove
 from tests.moto_mock_org.moto_models import SmallOrg
 
+cove = functools.partial(cove, thread_workers=1, raise_exception=True)
```

Comment:
In real use cases I almost always use the default settings for `thread_workers` and `raise_exception`.

But setting them like this makes the tests easier to read, write and debug.

* We can assume that `ExceptionDetails` and `FailedAssumeRole` are empty and just check `Results`.
* We can use `pytest.raises` to catch any exception.
* We can set a breakpoint in threaded code and get just one debugger instance.

I'd like to refactor all the tests to use this setup.

I'd rewrite `test_exceptions` to check that `ExceptionDetails` and `FailedAssumeRole` are set correctly for a few examples when `raise_exception` is `False`.

I would also add a test for the threading.